### ISSUE TITLE
Use options object instead of individual parameters

### ISSUE THREAD

**Issue by moritzebeling** [encoding-group/paraport#7]
What settings I might want to pass:

- element selector
- default speed
- speed factor to multiply with (to in/decrease speed of the whole animation at once)
- X or Y axis
- wether to issue events or not
- class name to be added when element is visible
- wether to animate or not (maybe I need events only)

possible object structure:
```js
let options = {
  selector: '.para', // string
  defaultSpeed: 2, // float
  multiplier: 1, // float
  axis: 'y', // string
  events: false, // bool
  visibleClass: 'para-visible', // string
  animate: true, // bool
}:
```

**Comment by reneweiser** [comment]
77e9a0157834e2d90749d5a516c88b7eb5941369

Created a branch for this feature and started to implement it. Most of the options are very straight forward but I'm not exactly sure what the options `events` and `animate` are supposed to do.

**Comment by moritzebeling** [comment]
let’s don't do **Y Axis** (at least for now).
It’s nice for a grown up plugin to have. But it would bloat the code and reduce simplicity and speed. Both things the current version shines at.

**Comment by moritzebeling** [comment]
Maybe overall, in this issue, let’s focus on passing the options as object – and not on added functionality.

So just:
```js
let options = {
  selector: '.para', // string
  defaultSpeed: 2, // float
};
```

Everything else can later be added as individual tasks, issues, branches...

**Comment by reneweiser** [comment]
Resolved by 958d9859a6539e23be0b81103c9fb4c7caaccfa9 - closed
